README.md

```diff
@@ -666,6 +666,9 @@ drive push -retry-count 4 a/bc/def terms
 the internal Google APIs use a value of 8MiB from constant `googleapi.DefaultUploadChunkSize`.
 Please note that your value has to be a multiple of and atleast the minimum  upload chunksize
 of 256KiB from constant `googleapi.MinUploadChunkSize`. See https://godoc.org/google.golang.org/api/googleapi#pkg-constants
+  If not set and the `-upload-rate-limit` is set, it will be the same as `-upload-rate-limit`.
```

Comment:
Perhaps could we make this

```
if `-upload-chunk-size` is not set yet `upload-rate-limit` is, `-upload-chunk-size` will be the same as `-upload-rate-limit`
```

It might seem like a bunch of repetition but exorcising `-upload-chunk-size` and `-upload-rate-limit` provide clarity

Reply:
ha, Okay.